Assets/BossRoom/Scripts/Game/Server/Entity/SpawnNetworkObjectAtLocation.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Game
+{
+    public class SpawnNetworkObjectAtLocation : NetworkBehaviour
```

Comment:
I feel like I've seen this before. For portals and previously for when we didn't have statically placed objects available.
Did we look into consolidating this (if possible of course)?